selfdrive/car/gm/values.py

```diff
@@ -47,6 +53,25 @@ class CAR:
   BUICK_REGAL = "BUICK REGAL ESSENCE 2018"
   ESCALADE_ESV = "CADILLAC ESCALADE ESV 2016"
 
+
+class Footnote(Enum):
+  OBD_II = CarFootnote(
+    "Requires an [OBD-II](https://comma.ai/shop/products/comma-car-harness) car harness and [community built ASCM harness]" +
+    "(https://github.com/commaai/openpilot/wiki/GM#hardware). NOTE: disconnecting the ASCM disables Automatic Emergency Braking (AEB).",
+    Column.MODEL)
+
+
+CAR_INFO: Dict[str, Union[CarInfo, List[CarInfo]]] = {
+  CAR.HOLDEN_ASTRA: CarInfo("Holden Astra 2017", "Adaptive Cruise"),
+  CAR.VOLT: CarInfo("Chevrolet Volt 2017-18", "Adaptive Cruise", footnotes=[Footnote.OBD_II], min_enable_speed=0),
```

Comment:
Trying to add the assert for the minEnableSpeed check. This is set in CarParams as 18 mph, however if you engage above that it will brake and accelerate from zero. Should it be 18 or 0 mph?